### DISCUSSION TITLE
Vanjaro working with 2SXC ?

### DISCUSSION THREAD

**Discussion by renderfarmer** [vanjarosoftware/Vanjaro.Platform#1755] [Q&A] [answered]
Has anyone tested if 2SexyContent works with Vanjaro?

**Comment by ashishpachauriap** [comment, marked as answer]
I have confirmed, It is working fine on Vanjaro.

**Comment by renderfarmer** [comment]
Can't seem to get it to work. It's always showing the "complete upgrade" button but it's not clickable. Also I have tested the ActionForm Moudule, but this only shows in View Mode, while in Edit Mode it shows only parts. Any tipps on working with 2SXC and ActionForm?

<img width="1898" alt="Bildschirmfoto 2022-02-11 um 15 43 44" src="https://user-images.githubusercontent.com/7823828/153612396-e1b2ab72-05af-48a2-8d2e-7953f4578864.png">

**Reply by Tychodewaard** [reply]
@renderfarmer for actionform, you need to CTRL-click the expand button. This opens ActionForm in a new tab. 
![image](https://user-images.githubusercontent.com/9091997/153614560-9e5de162-e0b9-45e5-8d02-2440523afaf6.png)

**Reply by Tychodewaard** [reply]
@renderfarmer  The complete upgrade is a 2SXC error. What I'd do is, drop the 2SXC on a page where you have the standard Xcillion. There you finish the upgrade.

**Reply by renderfarmer** [reply]
Hi @Tychodewaard , how do you create a page with the Xcillion Theme on a Vanjaro install? Can't seem to figure this out. I have copied the Xcillion to the Default Skins folder.

**Reply by Tychodewaard** [reply]
@renderfarmer 
I do not have experience with a Vanjaro install without DNN.
If you install DNN and after that Vanjaro, there is always /admin. This opens up the usual DNN Xcillion and persona bar. And the usual UI to create a DNN page.

**Reply by renderfarmer** [reply]
@Tychodewaard I still haven't managed to get 2SXC to work on Vanjaro (as install on top of DNN). I basically cannot access the module navigation. If I save the page, go to view the page and come back to edit, I get an error. Any idea on how this could work?
![Bildschirmfoto 2022-07-19 um 13 48 44](https://user-images.githubusercontent.com/7823828/179753441-b549fc6a-7310-4db2-b08f-e84057fa04b2.png)
![Bildschirmfoto 2022-07-19 um 14 44 52](https://user-images.githubusercontent.com/7823828/179753465-8c39a819-59bf-440a-80e4-ebe45d879af3.png)

**Reply by renderfarmer** [reply]
Thanks @meetmandeep for clearing this up. So this is how it works.

- Place App Module
- Save the page
- Exit the Vanjaro UX Manager
- Module is fully accessable
- Currently the module buttons do not work but everything can be accessed through 2SXC inner navigation

![Markup on 2022-07-20 at 15:00:42](https://user-images.githubusercontent.com/7823828/179988273-b508a926-3230-42f1-8654-c7d408ff3f07.png)

**Reply by Gaganbird** [reply]
It has been fixed in https://github.com/vanjarosoftware/Vanjaro.Platform/pull/2239
